jps-plugin/jps-plugin.iml

```diff
@@ -5,9 +5,8 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="IntelliJ IDEA IU-163.15188.11" jdkType="IDEA JDK" />
```

Comment:
And, I'm sorry for missing this the first time around.  Delete this file and I'll merge the PR.